### ISSUE TITLE
[announce] Mastodon posts a ? character instead of emoji

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1070]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Configure the Mastodon announcer and explicitly set the status with a leading emoji, such as
```
announce:
  twitter:
    active: RELEASE
    status: 🚀 {{projectName}} {{projectVersion}} has been released! {{releaseNotesUrl}}
  mastodon:
    active: RELEASE
    host: https://mastodon.social
    status: 🚀 {{projectName}} {{projectVersion}} has been released! {{releaseNotesUrl}}
```
2. Announce the project

### Expected Behaviour

Status posted to Mastodon with an emoji

### Actual Behaviour

A `?` appears instead of the emoji

```
[INFO]  Announcing release
[DEBUG]   [article] disabled. Skipping
[DEBUG]   [discord] disabled. Skipping
[DEBUG]   [discourse] disabled. Skipping
[DEBUG]   [discussions] disabled. Skipping
[DEBUG]   [gitter] disabled. Skipping
[DEBUG]   [googlechat] disabled. Skipping
[DEBUG]   [http] disabled. Skipping
[DEBUG]   [mail] disabled. Skipping
[INFO]    [mastodon] tooting: ? Kordamp Maven Parent 1.4.0 has been released! https://github.com/kordamp/kordamp-maven-parent/releases/tag/v1.4.0
[DEBUG]   [mastodon] status is 115 chars long
[DEBUG]   [mastodon] dry-run set to false
[DEBUG]   [mattermost] disabled. Skipping
[DEBUG]   [sdkman] disabled. Skipping
[DEBUG]   [slack] disabled. Skipping
[DEBUG]   [teams] disabled. Skipping
[DEBUG]   [telegram] disabled. Skipping
[INFO]    [twitter] tweeting: 🚀 kordamp-maven-parent 1.4.0 has been released! https://github.com/kordamp/kordamp-maven-parent/releases/tag/v1.4.0
[DEBUG]   [twitter] status is 116 chars long
[DEBUG]   [twitter] dry-run set to false
[DEBUG]   [webhooks] disabled. Skipping
[DEBUG]   [zulip] disabled. Skipping
```

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.3.1
- **JDK Version:** N/A

cc: @TomCools

**Comment by TomCools** [comment]
Oh.. it's fine on Twitter.. .weird. 😊 Narrows it down a lot.

**Comment by aalmiray** [comment]
Released in v1.4.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.4.0
